autogtp/Game.h

```diff
@@ -42,6 +42,7 @@ class Game : QProcess {
     QString getMove() const { return m_moveDone; }
     QString getFile() const { return m_fileName; }
     bool setMove(const QString& m);
+    bool checkGameEnd() { return (m_resignation || m_passes > 1 || m_moveNum > (19 * 19 * 2)); }
```

Comment:
this is longer than 80 characters, consider splitting onto three lines (and potentially moving to .cpp)